podbuilder/podbuilder.go

```diff
@@ -310,6 +311,14 @@ func (pb *PodBuilder) newPod() *corev1.Pod {
 		})
 	}
 
+	pb.run.Env = append(pb.run.Env, corev1.EnvVar{
+		Name:  "POD_TIMEOUT",
+		Value: strconv.FormatInt(int64(pb.test.Spec.TimeoutSeconds), 10)})
```

Comment:
Does it mean the pod would timeout after 10s?

Reply:
> Does it mean the pod would timeout after 10s?

Good point. strconv.Itoa() might be less confusing here.

Reply:
Or perhaps just fmt.Sprint(), which would bypass the conversion to int64.

Reply:
+1 to @paulosjca feedback. I think `fmt.Sprintf("%d", pb.test.Spec.TimeoutSeconds)` is cleanest.

Reply:
Thank you! I was not sure which to use, thank you for helping here. :)